labscript_devices/DummyPseudoclock/runviewer_parsers.py

```diff
@@ -0,0 +1,65 @@
+import labscript_utils.h5_lock  # noqa: F401
+import h5py
+import numpy as np
+
+
+class DummyPseudoclockParser(object):
+    clock_resolution = 25e-9
```

Comment:
This mimics the structure of `labscript_devices.PineBlaster` but could/should probably be extracted from connection table properties.